packages/client-core/src/user/menus/ProfileMenu.tsx

```diff
@@ -0,0 +1,692 @@
+              onChange={() => checkedTOS.set((v) => !v)}
+              label={t('user:usermenu.profile.agreeTOS')}
+            />
+            <Checkbox
+              checked={checked13OrOver.value}
+              onChange={() => checked13OrOver.set((v) => !v)}
+              label={t('user:usermenu.profile.confirmAge13')}
+            />
+          </>
+        )}
+
+        {!isGuest && !originallyAgeVerified.value && (
+          <Checkbox
+            checked={checked18OrOver}
+            onChange={submitAgeVerified}
+            label={t('user:usermenu.profile.confirmAge18')}
+          />
+        )}
+
+        <Input
+          value={username.value || ('' as UserName)}
+          state={errorUsername.value ? 'error' : undefined}
+          helperText={errorUsername.value}
+          onChange={handleUsernameChange}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter') updateUserName(e)
+          }}
+          labelProps={{
+            text: t('user:usermenu.profile.lbl-username'),
+            position: 'top'
+          }}
+          endComponent={
+            <button className="h-4 w-4" onMouseDown={updateUserName}>
+              <CheckLg />
+            </button>
+          }
+          fullWidth
+        />
+
+        {showUserId.value && (
+          <Input
+            labelProps={{
+              text: t('user:usermenu.profile.userIcon.userId'),
+              position: 'top'
+            }}
+            value={userId}
+            endComponent={
+              <button
+                className="h-4 w-4"
+                onMouseDown={() => {
+                  navigator.clipboard.writeText(userId)
+                  NotificationService.dispatchNotify(t('user:usermenu.profile.userIdCopied'), {
+                    variant: 'success'
+                  })
+                }}
+              >
+                <Copy03Lg />
+              </button>
+            }
+            fullWidth
+          />
+        )}
+
+        {showApiKey.value && (
+          <Input
+            labelProps={{
+              text: t('user:usermenu.profile.apiKey'),
+              position: 'top'
+            }}
+            value={apiKey?.token}
+            startComponent={
+              <button className="h-4 w-4" onMouseDown={refreshApiKey}>
+                <Refresh1Lg />
+              </button>
+            }
+            endComponent={
+              <button
+                className="h-4 w-4"
+                onMouseDown={() => {
+                  navigator.clipboard.writeText(apiKey?.token)
+                  NotificationService.dispatchNotify(t('user:usermenu.profile.apiKeyCopied'), {
+                    variant: 'success'
+                  })
+                }}
+              >
+                <Copy03Lg />
+              </button>
+            }
+            fullWidth
+          />
+        )}
+
+        {!hideLogin && acceptedTOS && isGuest && enableConnect && (
+          <>
+            <Input
```

Comment:
Missing onChange event in this input which is preventing me from logging in.

Reply:
Fixed